### DISCUSSION TITLE
.env.填写QQ频道机器人的信息后无法启动

### DISCUSSION THREAD

**Discussion by HX-Wrdzgzs** [nonebot/discussions#536] [问答] [answered]
![Screenshot_20241126_172040_com_microsoft_rdc_androidx_SessionActivity](https://github.com/user-attachments/assets/ac1d145e-6af0-41db-a603-d9418371a0e7)
![Screenshot_20241126_171853_com_microsoft_rdc_androidx_SessionActivity_edit_354945963282097](https://github.com/user-attachments/assets/95735335-7b3e-49dd-87c7-261011d14354)
根据_https://github.com/nonebot/adapter-qq?tab=readme-ov-file#qq_bots_中，P1也是按照github上填写了，但是错误原因在P2 看样子是QQ频道机器人信息填写错了吗？

**Comment by yanyongyu** [comment]
你的 c2c group那一行最后少了逗号

**Reply by HX-Wrdzgzs** [reply]
谢谢您的回答，目前登录问题已解决
想再问一下开发者，这种问题是插件问题吗？
![Screenshot_20241126_212709_com_microsoft_rdc_androidx_SessionActivity](https://github.com/user-attachments/assets/fabfc830-c6c7-4a7c-ad48-7a934f2b6083)

**Reply by yanyongyu** [reply, marked as answer]
这个插件不支持QQ适配器，仅支持onebot v11

**Reply by HX-Wrdzgzs** [reply]
好的，谢谢您
